core/src/apps/cardano/sign_tx.py

```diff
@@ -316,7 +293,42 @@ def _validate_metadata(metadata: Optional[bytes]) -> None:
         raise INVALID_METADATA
 
 
-def _serialize_tx(keychain: seed.Keychain, msg: CardanoSignTx) -> Tuple[bytes, bytes]:
+def _get_signed_tx_chunks(
+    ctx: wire.Context, cborized_tx: CborizedSignedTx, tx_hash: TxHash
```

Comment:
The `ctx` parameter is actually unused now :wink: `make pystyle` doesn't pick it up, nice.